dvc/stage.py

```diff
@@ -704,6 +703,7 @@ def check_can_commit(self, force):
                 )
             self.save()
 
+    @slow_link_guard
```

Comment:
So now it will print a warning for each stage that it checks out? :slightly_smiling_face: